mesh/core/src/main/protobuf/resolver.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package io.linkerd.mesh;
+
+import "dtab.proto";
+
+/**
+ * Supports resolution of a concrete name to a replica set.
+ *
+ * Status: Experimental; this API is not yet finalized.
+ */
+service Resolver {
+
+  rpc GetReplicas (ReplicasReq) returns (Replicas) {}
```

Comment:
"replicas" is a new piece of terminology we don't use elsewhere.  Likewise, in the existing namerd API, "resolve" means to go from a logical name directly to an address set.  The existing nomenclature isn't sacred but we should be deliberate about terminology.